index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Website</title>
+  </head>
+  <body>
+    <header>
+      <a class="name">Muh Noman</a>
+      <nav>
+        <img class="navi" src="/images/navi.png" alt="navigation" />
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h1>Hey There. I’m Noman</h1>
+        <h6>I’am a Software Developer</h6>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
```

Comment:
- [ ] You should use the `Poppins` font-family as suggested by previous reviewer and the Figma design. Please the `freecodecamp` link in previous comment to refer how to add google fonts.

Reply:
I Applied these changes now thanks for your suggestion